common/djangoapps/entitlements/api/v1/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url, patterns, include
+from rest_framework.routers import DefaultRouter
+
+from .views import EntitlementViewSet
+
+router = DefaultRouter()
+router.register(r'entitlements', EntitlementViewSet, base_name='entitlements')
+
+urlpatterns = patterns(
```

Comment:
Update for Django 1.11 compatibility. See https://openedx.atlassian.net/wiki/spaces/PLAT/pages/275349592/Django+1.11+URLs+Best+Practices.